.github/workflows/ephemeral-staging-teardown.yml

```
@@ -0,0 +1,105 @@
+  contents: read
+  pull-requests: read
+
+jobs:
+  destroy-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v4
+
+      - name: "Authenticate with GCP"
+        if: ${{ steps.check-labels.outputs.is_ephemeral_staging == 'true' }}
```

actionlint (1.7.4)

property "check-labels" is not defined in object type {}

(expression)